ui/components/ui/survey-toast/survey-toast.test.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
```

Comment:
We should create an E2E with a mocked API endpoint to ensure the survey displays when it should.

Reply:
I added it as a unit test, that mocks the return value and checks that the toast is rendered/not rendered